azext_iot/sdk/digitaltwins/controlplane/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "2022-10-31"
+VERSION = "2023-02-27-preview"
```

Comment:
Is there a reason this isn't `2023-01-31` ?

Reply:
the feature to be added (aka improved data history) is in the preview api

Reply:
used wrong version when autorest. Fixed